main/test/ca/uwaterloo/flix/library/TestPrelude.flix

```diff
@@ -156,3 +156,37 @@ def forwardPipe02(): Bool =
     let f = match (x, y) -> x + y;
     let r = (1, 2) |> f;
         r == 3
+
+/////////////////////////////////////////////////////////////////////////////
+// !>                                                                      //
+/////////////////////////////////////////////////////////////////////////////
+
+@test
+def name01(): Bool = region r {
```

Comment:
@magnus-madsen  I'm not sure, what the name of the tests should be

Reply:
Let's call it "imperativePipe".